src/services/chat/prepopulated.js

```diff
@@ -8,25 +8,24 @@ module.exports = async (id, users) => {
 
   console.log('prepopulated child function is called');
   try {
-    let userService = new UserService();
-    let ownUser = await userService.getUserById(id);
+    const userService = new UserService();
+    const ownUser = await userService.getUserById(id);
 
-    let res = await Promise.all(
+    const res = await Promise.all(
       users.map(async (user) => {
         const channelName = [];
         channelName.push(user.username);
         channelName.push(ownUser.username);
-        let generatedChannelId = generateRandomId();
-        console.log('generatedChannelId');
-        console.log(generatedChannelId);
+        const generatedChannelId = generateRandomId();
 
-        let chat = serverClient.channel('messaging', generatedChannelId, {
+        const chat = serverClient.channel('messaging', {
           name: channelName.join(', '),
           type_channel: 0,
-          created_by_id: ownUser.user_id
+          members: [id, user?.user_id],
+          created_by_id: id
         });
 
-        let status = await chat.create();
+        const status = await chat.create();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk are mostly about replacing `let` with `const` for variable declarations, which is a good practice when the variable is not going to be reassigned. This can help prevent bugs due to accidental reassignment.

However, there's an issue on line 21 where the channel ID is no longer being used in the `serverClient.channel` function call. The original code passed `generatedChannelId` as the second argument to this function, but the new code does not. If the generated channel ID is necessary for creating a unique chat channel, this could be a problem.

```diff
-        let chat = serverClient.channel('messaging', generatedChannelId, {
+        const chat = serverClient.channel('messaging', {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->